.github/workflows/static-checks.yaml

```diff
@@ -94,3 +94,27 @@ jobs:
       if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
       run: |
         cd ${GOPATH}/src/github.com/${{ github.repository }} && sudo -E PATH="$PATH" make test
+
+  test-dragonball:
+    runs-on: self-hosted
+    env:
+      RUST_BACKTRACE: "1"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set env
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
+        run: |
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+      - name: Install Rust
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
+        run: |
+          ./ci/install_rust.sh
+          PATH=$PATH:"$HOME/.cargo/bin"
+      - name: Run Unit Test
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
+        run: |
+          cd src/dragonball
+          /root/.cargo/bin/cargo version
+          rustc --version
+          # TODO: Using the cargo command directly will get `sudo: cargo: command not found` error.
```

Comment:
Add `PATH=$PATH:"$HOME/.cargo/bin"` as line 112?

Reply:
It's not work.
I have tried: `PATH=$PATH:"$HOME/.cargo/bin"` and `sudo -E PATH="$PATH" cargo test`
See 
https://github.com/kata-containers/kata-containers/actions/runs/3218377679/jobs/5262478984
https://github.com/kata-containers/kata-containers/actions/runs/3218316683/jobs/5262340478
for more details.